include/circt/Dialect/FIRRTL/FIRRTLExpressions.td

```diff
@@ -919,6 +919,18 @@ def HasBeenResetIntrinsicOp : FIRRTLOp<"int.has_been_reset", [Pure]> {
   let assemblyFormat = "$clock `,` $reset attr-dict `:` type($reset)";
 }
 
+def FPGAProbeIntrinsicOp : FIRRTLOp<"int.fpga_probe", []> {
+  let summary = "FPGA ILA probe";
+
+  let description = [{
+      Result of `firrtl.int.fpga_probe` for probing signals
+  }];
```

Comment:
It looks like most of the comments want to avoid the Xilinx-specific "ILA" term. How about something like the following:
```suggestion
  let summary = "Mark a value to be observed through FPGA debugging facilities";

  let description = [{
    The `firrtl.int.fpga_probe` intrinsic marks a value in the IR to be made observable
    through FPGA debugging facilities. Most FPGAs offer a form of signal observation
    or logic analyzer to debug a design. This operation allows the IR to indicate which
    signals should be made observable for debugging. Later FPGA-specific passes
    may then pick this information up and materialize the necessary logic analyzers
    or tool scripts.
  }];
```

Reply:
makes sense thanks!